src/views/mfa-verify/SendEmailAndVerifyCodeForm.js

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) 2015-2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'views/shared/TextBox',
+  'views/ResendEmailView'
+], function (Okta, TextBox,  ResendEmailView) {
+
+  const _ = Okta._;
+  const createEmailMaskElement = function () {
+    const email = this.model.get('email');
+    const emailTpl = Okta.tpl('<span class="mask-email">{{email}}</span>');
+    return {factorEmail: emailTpl({email})};
+  };
+
+  const VerifyEmailCodeForm = Okta.Form.extend({
+    layout: 'o-form-theme',
+    className: 'mfa-verify-email',
+    title: _.partial(Okta.loc, 'email.mfa.title', 'login'),
+    noButtonBar: false,
+    autoSave: true,
+    noCancelButton: true,
+    attributes: {
+      'data-se': 'factor-email',
+    },
+    save: function () {
+      return this.options.appState.get('isMfaChallenge')
+        ? Okta.loc('mfa.challenge.verify', 'login')
+        : Okta.loc('email.button.send', 'login');
+    },
+
+    events: Object.assign({}, Okta.Form.prototype.events, {
```

Comment:
curious, did `events: { submit: .... }` not work for some reason? Or is there anything in parent Form that we need here?

Reply:
I think it would work at this case although it removes [other events](https://github.com/okta/okta-ui/blob/master/packages/courage/src/views/forms/BaseForm.js#L65) defined in the base form.
In general when I do override, I tend to be explicit and keep whatever else comes from Parent.